src/shared/components/oncoprint/DataUtils.ts

```diff
@@ -123,13 +133,19 @@ export function fillGeneticTrackDatum(
                 }
                 break;
             case "MUTATION_EXTENDED":
+
                 let oncoprintMutationType = getOncoprintMutationType(getSimplifiedMutationType(event.mutationType)!);
+
                 if (oncoprintMutationType === "fusion") {
                     dispFusion = true;
                 } else {
                     if (event.putativeDriver) {
                         oncoprintMutationType += "_rec";
                     }
+                    if (event.mutationStatus === 'Germline') {
```

Comment:
why do we need this if you're already tracking germline status with `disp_germline`?